src/lib/accessibility/Behaviors/Menu/menuItemBehavior.ts

```diff
@@ -1,40 +1,57 @@
 import { Accessibility } from '../../types'
 import { IS_FOCUSABLE_ATTRIBUTE } from '../../FocusZone/focusUtilities'
 import * as keyboardKey from 'keyboard-key'
+import * as _ from 'lodash'
 
 /**
  * @description
  * The behavior is designed for particular structure of menu item. The item consists of root element and anchor inside the root element.
  *
  * @specification
- * Adds role 'presentation' to 'root' component's part.
- * Adds role 'menuitem' to 'anchor' component's part.
- * Adds attribute 'tabIndex=0' to 'anchor' component's part.
- * Adds attribute 'data-is-focusable=true' to 'anchor' component's part.
- * Adds attribute 'aria-label' based on the property 'aria-label' to 'anchor' component's part.
- * Adds attribute 'aria-labelledby' based on the property 'aria-labelledby' to 'anchor' component's part.
- * Adds attribute 'aria-describedby' based on the property 'aria-describedby' to 'anchor' component's part.
+ * Adds role 'presentation' to 'wrapper' component's part.
+ * Adds role 'menuitem' to 'root' component's part.
+ * Adds attribute 'tabIndex=0' to 'root' component's part.
+ * Adds attribute 'data-is-focusable=true' to 'root' component's part.
+ * Adds attribute 'aria-label' based on the property 'aria-label' to 'root' component's part.
+ * Adds attribute 'aria-labelledby' based on the property 'aria-labelledby' to 'root' component's part.
+ * Adds attribute 'aria-describedby' based on the property 'aria-describedby' to 'root' component's part.
+ * Adds attribute 'aria-expanded=true' based on the property 'menuOpen' if the component has 'menu' property to 'root' component's part.
+ * Adds attribute 'aria-haspopup=true' to 'root' component's part based on the property 'menu'.
+ * Performs 'performClick' action with 'Enter' or 'Spacebar' on 'wrapper'.
```

Comment:
would suggest to rename `Performs` to `Triggers`, as it will better express what behavior is actually doing. Also, it will aid the readability of the spec, especially for this particular line :)